product_configurator/models/product_config.py

```diff
@@ -553,21 +554,16 @@ def update_session_configuration_value(self, vals, product_tmpl_id=None):
 
             # Add attribute values from the client except custom attribute
             # If a custom value is being written, but field name is not in
-            #   the write dictionary, then it must be a custom value!
+            # the write dictionary, then it must be a custom value!
             if vals.get(field_name, custom_val.id) != custom_val.id:
                 if attr_line.multi and isinstance(vals[field_name], list):
-                    if not vals[field_name]:
-                        field_val = None
-                    else:
-                        field_val = vals[field_name][0][2]
+                    field_val = self._update_field_values(vals, field_name, attr_line)
                 elif not attr_line.multi and isinstance(vals[field_name], int):
                     field_val = vals[field_name]
                 else:
                     raise UserError(
-                        _(
-                            "An error occurred while parsing value for attribute %s",
-                            attr_line.attribute_id.name,
-                        )
+                        _("An error occurred while parsing value for attribute %s")
+                        % attr_line.attribute_id.name
```

Comment:
This was better before changing 😓 , using gettext's parameters
Could you revert it ? 🙏🏻 


```suggestion
                        _(
                            "An error occurred while parsing value for attribute %s",
                            attr_line.attribute_id.name
                        )
```

Reply:
The pre-commit or black format in the IDE played a role, I have changed back.